### ISSUE TITLE
Issue from update from 1.7.2 to 1.7.3 

### ISSUE THREAD

**Issue by Doctor-Who** [eea/copernicus_quality_tools#9]
Hello,

When trying updating as usually by editing tag in my Docker-Compose, the 1.7.3 returned me an error when trying to open the URL : 

```
Request Method: | GET
-- | --
http://process01.sirs-fr.com:8000/
3.0.7
TemplateSyntaxError
'staticfiles' is not a registered tag library. Must be one of: admin_list admin_modify admin_urls cache i18n l10n log static tz
/usr/local/lib/python3.7/dist-packages/django/template/defaulttags.py in find_library, line 1025
/usr/bin/python3
3.7.5
['/usr/local/src/copernicus_quality_tools/src/qc_tool/frontend',  '/usr/local/src/copernicus_quality_tools/src',  '/usr/lib/python37.zip',  '/usr/lib/python3.7',  '/usr/lib/python3.7/lib-dynload',  '/usr/local/lib/python3.7/dist-packages',  '/usr/lib/python3/dist-packages']
Wed, 10 Jun 2020 07:01:33 +0200
```

![Screenshot_20200610_094049](https://user-images.githubusercontent.com/5890416/84240582-8bace380-aafe-11ea-9431-c8d782175147.png)

Regards,

Nicolas

**Comment by jirikadlec2** [comment]
Hello,
The issue was caused by upgrading Django library to Django 3.0 in QC tool 1.7.3 which we did not completely check. 
We have fixed the issue in today's QC tool release [1.7.4](https://github.com/eea/copernicus_quality_tools/releases/tag/1.7.4)

Please upgrade to 1.7.4.
Regards,

Jiri

**Comment by Doctor-Who** [comment]
Work like a charm, thanks :+1:
